reference/javascript/sidebar.html

```diff
@@ -38,6 +38,12 @@
       <li>
         <a href="#donationservice-completebbspdonation">completeBBSPDonation</a>
       </li>
+	   <li>
+        <a href="#donationservice-getRecurringGiftInstallmentAmount">getRecurringGiftInstallmentAmount</a>
+      </li>
+	   <li>
+        <a href="#donationservice-getRecurringGiftLastPaymentDate">getRecurringGiftLastPaymentDate</a>
```

Comment:
This link is case-sensitive to the `id` used in the corresponding section, so it should be `<a href="donationservice-getrecurringgiftlastpaymentdate"> getRecurringGiftLastPaymentDate</a>`.